modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/service/impl/BatchEngineExportTaskServiceImpl.java

```diff
@@ -30,6 +30,25 @@
 public class BatchEngineExportTaskServiceImpl
 	extends BatchEngineExportTaskServiceBaseImpl {
 
+	@Override
+	public BatchEngineExportTask getBatchEngineExportTask(Long exportTaskId)
```

Comment:
The parameter in the DB is called `batchEngineExportTaskId`. I think we should use that name